e2e-tests/resources/authd/utils.resource

```diff
@@ -0,0 +1,143 @@
+*** Settings ***
+Documentation       authd test resources
+
+Resource            kvm.resource
+Library             ./StringUtils.py    AS    StringUtils
+Library             ./VideoLogger.py    AS    VideoLogger
+Library             ./Snapshot.py    AS    Snapshot
+Library             ./SSH.py    AS    SSH
+Library             Process
+Library             OperatingSystem
+
+
+*** Keywords ***
+Log In
+    Match Text    Not listed    300
+    Hid.Keys Combo    Return
+    BuiltIn.Sleep    2
+    Hid.Type String    ubuntu
+    BuiltIn.Sleep    2
+    Hid.Keys Combo    Return
+    Wait Until Desktop Ready
+
+
+Log Out
+    Run Command    gnome-session-quit --logout --no-prompt
+
+
+Wait Until Desktop Ready
+    Wait Until Keyword Succeeds    1 min    1 sec
+    ...    Try Desktop Ready
+
+
+Try Desktop Ready
+    Hid.Move Pointer To Proportional    1    1
+    BuiltIn.Sleep    1
+    Hid.Move Pointer To Proportional    0    1
+    Match Text    Show Apps    10
+
+
+Open Terminal
+    Run Command    x-terminal-emulator
+    Hid.Move Pointer To Proportional    1    1
+    Match Text    @ubuntu:~$    30
+    Hid.Keys Combo    F11
+    Hid.Move Pointer To Proportional    1    1
+    Match Text    @ubuntu:~$    30
+
+
+Open Terminal In Sudo Mode
+    Open Terminal
+    Enter Sudo Mode In Terminal
+
+
+Enter Sudo Mode In Terminal
+    Hid.Type String    clear
+    Hid.Keys Combo    Return
+    Hid.Type String    sudo -s
+    Hid.Keys Combo    Return
+    Match Text    root@ubuntu    30
+    Hid.Type String    clear
+    Hid.Keys Combo    Return
+    Match Text    root@ubuntu    30
+
+
+Close Focused Window
+    Hid.Keys Combo    Alt_L   F4
+
+
+Close Terminal In Sudo Mode
+    Hid.Type String    clear
+    Hid.Keys Combo    Return
+    Match Text    root@ubuntu    30
+    Hid.Type String    exit
+    Hid.Keys Combo    Return
+    Close Focused Window
+
+
+Log Out From Terminal Session
+    Match Text    @ubuntu    30
+    Hid.Type String    exit
```

Comment:
This doesn't have any effect without a subsequent `Hid.Keys Combo    Return`. It doesn't seem to be needed though, the `]]]` is enough to exit the machinectl login session, so I would just remove it.

Reply:
done